les/peer.go

```diff
@@ -976,6 +992,30 @@ func (ps *peerSet) AllPeers() []*peer {
 	return list
 }
 
+//Ray: Get all peers that only support les and have no other caps.
```

Comment:
I just found a better way to make this check. A peer is a client if `p.fcServer != nil` https://github.com/celo-org/celo-blockchain/blob/7d6b3c511560f66e94ae4d3ae2ea903975bde569/les/peer.go#L115